lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -258,6 +289,26 @@ def self.available_options
                                  is_string: false,
                              default_value: false),
 
+          FastlaneCore::ConfigItem.new(key: :mapping,
+                                  env_name: "APPCENTER_DISTRIBUTE_ANDROID_MAPPING",
+                               description: "Path to your Android Proguard or R8 mapping.txt",
+                                  optional: true,
+                                      type: String,
+                              verify_block: proc do |value|
+                                accepted_formats = ["mapping.txt"]
```

Comment:
Let's not restrict to only `mapping.txt` input

Reply:
Agree, when we talked about this last, this was mostly in the context of a CI build running, or uploading the results of a CI build, in which case the file we're looking for is named "mapping.txt" - but in the case of both CLI and Fastlane (see comment in https://github.com/microsoft/appcenter-cli/pull/594 as well), it is possible to run the upload from a different location. 
So we should not restrict the file name for the mapping file to only mapping.txt - we can still check for the .txt extension. Also, when uploading to Diagnostics, we can specify the filename - it would be great if we can use the base name of the mapping file here, so this does not confuse users.